### DISCUSSION TITLE
Is there a way to make multi-window desktop app?

### DISCUSSION THREAD

**Discussion by rlhk** [flet-dev/flet#3828] [Q&A]
### Question

I need a way to make multi-window app on macOS. Looks like Flutter itself does support such feature. Searched a bit and didn't find a solution. Is there a plan for it?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
This is not yet possible in Flet.

>  Looks like Flutter itself does support such feature. 

Can you share a link?

**Reply by rlhk** [reply]
I first found this
https://pub.dev/packages/desktop_multi_window/example

Then there seems some issue re Flutter dev
https://www.reddit.com/r/FlutterDev/comments/1ch4xbm/flutter_desktop_multi_window_paused/
https://github.com/flutter/flutter/issues/142845

Quite confusing TBH ...
